lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3261,9 +3260,11 @@ bool SPIRVToLLVM::translate() {
   }
 
   for (unsigned I = 0, E = BM->getNumFunctions(); I != E; ++I) {
-    transFunction(BM->getFunction(I));
+    transUserSemantic(I);
```

Comment:
I think that this change breaks readability of the code: previously we translated functions here, but now it might seem as we don't translate them at all. I do understand that this happens implicitly now, but I recommend  to have 2 separate calls here:

```c++
transFunction()
transUserSemantic()
```